README.md

```diff
@@ -157,7 +157,7 @@ The main data flow is represented in blue. Data samples are selected by the Data
 
 ## DPL
 
-TODO -> piotr
+[Data Processing Layer](https://github.com/AliceO2Group/AliceO2/blob/dev/Framework/Core/README.md) is a software framework developed as a part of O2 project. It structurizes the computing into units called _Data Processors_ - processes that communicate with each other via messages. DPL takes care of generating and running the processing topology out of user declaration code, serializing and deserializing messages and much more. Each piece of data is characterized by its `DataHeader`, which consists (among others) of `dataOrigin`, `dataDescription` and `SubSpecification` - for example `{"MFT", "TRACKS", 0}`.
```

Comment:
you could maybe add a sentence about the fact that DPL provides the data processors with all the pieces for a given timestamp. I don't think it needs to be more detailed otherwise.